### ISSUE TITLE
The player is not affected by weather radiation.

### ISSUE THREAD

**Issue by Scrivener07** [Scrivener07/FO4_GasMask#30]
With True Storms activated and possibly without once #29 is resolved, weather that should affect the player with radiation does not work correctly.

**Comment by pbrown24** [comment]
I have true storms and am affected by radiation, maybe your load order?

**Comment by Scrivener07** [comment]
![issue_30_01](https://cloud.githubusercontent.com/assets/1912967/19617409/96f3863a-97e4-11e6-8e63-9bee00e260e2.PNG)
The only mods in my game folder is Armor Keywords, True Storms, and Gas Masks of the Wasteland.
Maybe there is another reason for this to happen. Another factor is that I tested by using `coc RedRocketExt`. At least in the case of [GasMask_Start](https://github.com/Scrivener07/FO4_GasMask/blob/9f98b36cd29d0ec04f851fd9940a806a33386d4a/Data/Scripts/Source/Fallout2287/GasMask_Start.psc) the MQ102 condition will be false. Maybe there are other startup checks else where in the code?

**Comment by pbrown24** [comment]
Yeah maybe, we might need some, or keep the holotapes abundantly accessible,

**Comment by Scrivener07** [comment]
Same issue as 2 days ago. This is my load order after True Storms was removed as a master.
![issue_30_2](https://cloud.githubusercontent.com/assets/1912967/19632801/24b3b49c-995f-11e6-92a8-4810b539608e.png)
